src/db/effects_tables_vco.tsv

```diff
@@ -16,3 +16,4 @@ vco_ogr_greasus_speed	battle_movement_character.png	101	battle_movement_characte
 vco_ogr_greasus_charge_bonus	charge_character.png	101	charge_character.png	campaign	true
 vco_ogr_greasus_armour	armour_character.png	101	armour_character.png	campaign	true
 vco_ogr_greasus_ward_save	resistance_ward_save.png	101	resistance_ward_save.png	campaign	true
+vco_skv_pooled_resource_skaven_food_victory	pooled_resource_food.png	101	pooled_resource_food.png	campaign	true
```

Comment:
W: I do not like the `vco_skv_pooled_resource_skaven_food_victory` becuase it is redudant to have a `pooled_resource` key to include `pooled_resource`. I would change it to `vco_skv_per_turn_food`. It would also be more coherent with the other pooled resource effects: https://github.com/msolefonte/vco/blob/main/src/db/effects_tables_vco.tsv#L15

Note that all these changes about keys can be solved with find and replace direclty in the TSVs.